tests/unit_tests/utils/test_general.py

```diff
@@ -790,11 +790,11 @@ def test_validate_data_type():
         ("str", "hello", None, False, True),
         ("bool", True, None, False, True),
         ("bool", 1, None, False, False),
-        ("int", None, type(5), False, True),
-        ("float", None, type(3.14), False, True),
-        ("str", None, type("hello"), False, True),
-        ("bool", None, type(True), False, True),
-        ("bool", None, type(True), False, True),
+        ("int", None, int, False, True),
+        ("float", None, float, False, True),
+        ("str", None, str, False, True),
+        ("bool", None, bool, False, True),
+        ("bool", None, bool, False, True),
         # Test allow_subtypes=True
```

Comment:
For a moment I thought this was by-passing the test, but I now think that this is fine. I Just commented just in case you disagree with these automatic changes.

Reply:
I've seen them and I think they are fine.

Reply:
Good!